slack-server-integration-common/src/main/java/com/atlassian/plugins/slack/api/client/interceptor/BackoffRetryInterceptor.java

```diff
@@ -47,21 +47,48 @@ public Response intercept(final Chain chain) throws IOException {
                 }
 
                 response = chain.proceed(request);
+                final int code = response.code();
                 if (response.isSuccessful()) {
+                    if (tryCount > 0 && log.isDebugEnabled()) {
+                        log.debug("Backoff recover req_id={} {} {} -> {} - attempt {}/{}",
```

Comment:
I have added recovering logs, so that we can know whether a retry worked or not.